src/meta_schedule/space_generator/post_order_apply.cc

```diff
@@ -106,20 +96,63 @@ class PostOrderApplyNode : public SpaceGeneratorNode {
 
   Array<tir::Schedule> GenerateDesignSpace(const IRModule& mod_) final {
     using ScheduleAndUnvisitedBlocks = std::pair<tir::Schedule, Array<tir::BlockRV>>;
-    tir::Schedule sch = tir::Schedule::Traced(        //
-        /*mod=*/mod_,                                 //
-        /*rand_state=*/ForkSeed(&this->rand_state_),  //
-        /*debug_mode=*/0,                             //
+    tir::Schedule sch = tir::Schedule::Traced(                          //
+        /*mod=*/mod_,                                                   //
+        /*rand_state=*/ForkSeed(&this->rand_state_),                    //
+        /*debug_mode=*/tir::kVerifySRefTree | tir::kVerifyCachedFlags,  //
```

Comment:
I think we better keep the code here without the workaround.

Reply:
Let's work around it for now, and get bugs fixed later